File: Java/src/patterns/factory/factoryMethod/PizzaTest.java
package patterns.factory.factoryMethod;

public class PizzaTest {
	public static void main(String argv[]) {
		PizzaStore store = new NewYorkPizzaStore();
		store.orderPizza("Chees");
		store.orderPizza("Clam");
		store = new ChicagoPizzaStore();
		store.orderPizza("Chees");
		store.orderPizza("Clam");
	}
}

File: cpp/src/patterns/factory/AbstractFactory/PizzaTest.cc
#include <iostream>
#include "PizzaStore.h"

int main () {
    PizzaStore *store = new NewYorkPizzaStore();
    store->OrderPizza("Cheess");
    store->OrderPizza("Clam");
    store = new ChicagoPizzaStore();
    store->OrderPizza("Cheess");
    store->OrderPizza("Clam");
    return 0;
}
